tests/ui/conftest.py

```diff
@@ -49,16 +101,161 @@ def jwt_secret(base_url, variables):
 
 
 @pytest.fixture
-def initial_data(transactional_db):
-    from olympia.amo.tests import addon_factory, user_factory
-    from olympia.addons.models import AddonUser
+def initial_data(transactional_db, pytestconfig):
+    """
+    Fixture used to fill database will dummy addons.
+
+    Creates exactly 10 random addons with users that are also randomly
+    generated.
+    """
+    if pytestconfig.option.usingliveserver:
```

Comment:
Alternatively, return early if this is false, which avoids the additional level of indentation. This applies to the following fixtures too.